aries-site/src/themes/aries.js

```diff
@@ -6,6 +6,14 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe.
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
+  pageHeader: {
+    large: {
+      columns: [['small', 'flex'], 'auto'],
+    },
+    xlarge: {
+      columns: [['medium', 'flex'], 'auto'],
```

Comment:
Needed to change the values for the anatomy we can have a separate `grid` for anatomy put this in here to view and see if we want to go that approach. The problem Im running into is even though the screen size is `large`  or `medium` the anatomy is not taking up the whole area due to the side nav and the way the page is laid out so I needed to shrink these values for the Title so the buttons are left with enough room not to wrap.

Reply:
@taysea

Reply:
I'm not following why we have this in `aries.js` -- is your thought that we will only adjust the pageHeader Grid definition for the anatomy diagram? I'm not following why we need something separate than what the other page header examples need.

Reply:
I had this here to demo the changes for the anatomy to make sure were on the same page then I can pull this out and make a new grid for the anatomy diagram. If we like these values

Reply:
took it out of `aries.js` and put theme.extend like we discussed